crs_support/urdf/crs.urdf.xacro

```diff
@@ -333,9 +333,9 @@
         </collision>
         <collision> Back Wall top beam
             <geometry>
-                <box size="0.1 0.15 3.0" />
+                <box size="1.25 0.12 3.0" />
```

Comment:
Are these specific to the SwRI workcell?  if so we should create a separate xacro

Reply:
This is not SwRI specific, it has additionally collision geometry that should be included in the Spirit workcell as well